changebot/blueprints/stale_pull_requests.py

```diff
@@ -19,26 +20,40 @@ def close_stale_pull_requests():
     return "All good"
 
 
-PULL_REQUESTS_CLOSE_WARNING = """
-Hi humans :wave: - this pull request hasn't had any new commits for approximately {pasttime}. **I plan to close this in {futuretime} if the pull request doesn't have any new commits by then.**
+PULL_REQUESTS_CLOSE_WARNING = re.sub('(\w+)\n', r'\1', """
+Hi humans :wave: - this pull request hasn't had any new commits for
+approximately {pasttime}. **I plan to close this in {futuretime} if the pull
+request doesn't have any new commits by then.**
 
-In lieu of a stalled pull request, please close this and open an issue instead to revisit in the future. Maintainers may also choose to add `keep-open` label to keep this PR open but it is discouraged unless absolutely necessary.
+In lieu of a stalled pull request, please consider closing this and open an
+issue instead if a reminder is needed to revisit in the future. Maintainers
+may also choose to add `keep-open` label to keep this PR open but it is
+discouraged unless absolutely necessary.
 
-*If you believe I commented on this issue incorrectly, please report this [here](https://github.com/astropy/astropy-bot/issues).*
-"""
+If this PR is still needed to be reviewed, as an author, you can rebase it
+to reset the clock. You may also consider to send a reminder e-mail about it
+to the [astropy-dev mailing list](http://groups.google.com/group/astropy-dev).
+
+*If you believe I commented on this pull request incorrectly, please report
+ this [here](https://github.com/astropy/astropy-bot/issues).*
+""").strip()
```

Comment:
You need to get rid of the wrapping here, so ``.replace(os.linesep, ' ')`` - also below

Reply:
Ah wait you did it with a regular expression, oops - any reason why not just use replace?

Reply:
As it then replaces all new lines, and everything ends up in on line.